tests/js/spec/views/dataExport/dataDownload.spec.jsx

```diff
@@ -29,35 +30,34 @@ describe('DataDownload', function() {
     const wrapper = shallow(<DataDownload params={mockRouteParams} />);
     expect(wrapper.state('download')).toEqual({status});
     expect(wrapper.state('download').dateExpired).toBeUndefined();
-    const contentWrapper = wrapper.find('ContentContainer');
-    expect(contentWrapper.children()).toHaveLength(3);
-    expect(contentWrapper.find('h3').text()).toBe("You're Early!");
+    expect(wrapper.find('Header').text()).toBe('What are you doing here?');
   });
 
   it("should render the 'Expired' view when appropriate", function() {
     const status = DownloadStatus.Expired;
-    getDataExportDetails({status});
-    const wrapper = shallow(<DataDownload params={mockRouteParams} />);
-    expect(wrapper.state('download')).toEqual({status});
+    const response = {status, query: {type: ExportQueryType.IssuesByTag}};
+    getDataExportDetails(response);
+    const wrapper = mountWithTheme(<DataDownload params={mockRouteParams} />);
+    expect(wrapper.state('download')).toEqual(response);
     expect(wrapper.state('download').dateExpired).toBeUndefined();
-    const contentWrapper = wrapper.find('ContentContainer');
-    expect(contentWrapper.children()).toHaveLength(3);
-    expect(contentWrapper.find('h3').text()).toBe('Sorry!');
+    expect(wrapper.find('Header').text()).toBe('This is awkward.');
+    const buttonWrapper = wrapper.find('a[aria-label="Start a New Download"]');
+    expect(buttonWrapper.prop('href')).toBe(
+      `/organizations/${mockRouteParams.orgId}/issues/`
+    );
   });
 
   it("should render the 'Valid' view when appropriate", function() {
     const status = DownloadStatus.Valid;
     getDataExportDetails({dateExpired, status});
     const wrapper = mountWithTheme(<DataDownload params={mockRouteParams} />);
     expect(wrapper.state('download')).toEqual({dateExpired, status});
-    const contentWrapper = wrapper.find('ContentContainer').childAt(0);
-    expect(contentWrapper.children()).toHaveLength(5);
```

Comment:
Removed any child length tests, it was pretty unnecessary in the first place to count `<p>` tags.